packages/clerk-js/src/ui/contexts/components/SignIn.ts

```
@@ -121,16 +120,25 @@
   const signUpContinueUrl = buildURL({ base: signUpUrl, hashPath: '/continue' }, { stringify: true });
 
-  const sessionTaskUrl = buildSessionTaskRedirectUrl({
-    task: clerk.session?.currentTask,
-    path: ctx.path,
-    routing: ctx.routing,
-    baseUrl: signInUrl,
-    taskUrls: clerk.__internal_getOption('taskUrls'),
-  });
+  console.log({ basePath, rest });
+  const navigateOnSetActive = async ({ session, redirectUrl }: { session: SessionResource; redirectUrl: string }) => {
+    debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)